### PR TITLE
529269 add icon to xtext build console

### PR DESCRIPTION
**Pull request by Bananeweizen**
Signed-off-by: Michael Keppler <Michael.Keppler@gmx.de>

### CODE REVIEW

**Comment by szarnekow** [comment]
I'm not an expert on this, but do we need to provide a HDPI icon?

**Comment by Bananeweizen** [comment]
Good catch. I must admit I don't really know myself, but I can add an @2 icon, to be on the safe side.

File: org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/debug/XtextBuildConsole.xtend [hunk 1]
```diff
@@ -24,7 +28,10 @@ class XtextBuildConsole extends IOConsole {
 	val PrintStream out
 
 	new() {
-		super("Xtext Build", "xtextBuildConsole", null, true)
+		super("Xtext Build", "xtextBuildConsole",
+			ImageDescriptor.createFromURL(
+				FileLocator.find(Platform.getBundle("org.eclipse.xtext.builder"), new Path("icons/console.png"), null)),
```
Comment (kthoms) [inline org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/debug/XtextBuildConsole.xtend]: `Activator.^default.bundle.getEntry("icons/console.png")` should do the job also

**Comment by Bananeweizen** [comment]
Thanks for your comment. I will try to remember avoiding that lookup for other situations. That's what happens if you just look at the FAQ for getting a bundle based image. :)

**Comment by kthoms** [comment]
Thank you, Michael!

**Pull request merged**